c7n/schema.py

```diff
@@ -26,6 +26,7 @@
 from c7n.provider import clouds
 from c7n.query import sources
 from c7n.resources import load_available
+from c7n.resources.aws import fake_session
```

Comment:
hmmm.. I'd like to try to limit moving aws specific imports into generic top level package modules, we're aiming to move the core modules out to their own package over time, so generally want to limit import flow the other way (from aws back into core). another place for this would be in the resources/aws.py module